src/infill/SubDivCube.cpp

```diff
@@ -252,11 +252,9 @@ void SubDivCube::rotatePointInitial(Point& target)
 
 void SubDivCube::rotatePoint120(Point& target)
 {
-    constexpr double sqrt_three_fourths = 0.8660254037844386467637231707529361834714026269051903; //!< sqrt(3.0 / 4.0) = sqrt(3) / 2
-    int64_t x;
-    x = (-0.5) * target.X - sqrt_three_fourths * target.Y;
-    target.Y = (-0.5)*target.Y + sqrt_three_fourths * target.X;
-    target.X = x;
+    constexpr double sqrt_three_fourths = sqrt(3.0 / 4.0);
+    target.Y = (-0.5) * target.Y + sqrt_three_fourths * target.X;
+    target.X = (-0.5) * target.X - sqrt_three_fourths * target.Y;
```

Comment:
The commit message of this commit is "Use coord_t instead of int64_t"

I've warned about this very mistake already while reviewing this PR earlier.

I believe it is very important to clearly indicate what changes you make - especially if most of the changes in a PR are only simple refactors which have a very small chance of introducing bugs. You should never commit high risk changes in a low risk changes commit and even more important: you should never commit changes and not describe those changes.

Always go through all changes before you commit them, e.g. by using `git add -p` instead of `git commit -a`. Stay sharp! 🦈

Reply:
Fixed here: e13b0a7415f6fc13c3aa3514aabb2d146c06ec40

I do my best to separate out commits properly. Enough to also get some comments about that I should've squashed some of them, sometimes. The problem with cases like this is that I boyscout them while making large replacements under time pressure, so if I discover five of these I don't take the time to separate all of them out to separate commits. Especially if I'm assuming that nobody is going to go through the 400 commits separately.

Reply:
Ever since my first review I've been going through the commits separately. That is: I skip all commits which sound trivial and pay more attention to a commit with a commit message which sounds more error-prone.

I sometimes do a lot of things in the same time as well. Then I try to commit in several commits, but sometimes it just becomes too much. Then I put something in the commit message like "A couple of small fixes." It's vague, but at least it's true. That way you are not hiding a 'hotfix' in a commit which is supposed to be about something else.